### ISSUE TITLE
Change "unstable" version of the spec to "unreleased"

### ISSUE THREAD

**Issue by ShadowJonathan** [matrix-org/matrix-doc#3685]
Currently, https://spec.matrix.org/unstable/ tracks `main`, while positioning itself as "unstable" with the URL, and with the snippet at the top of the pages;

```
You're looking at an unstable version of this specification. Unstable specifications may change at any time without notice.
```

I believe this wording/naming can be softened to unreleased, as often enough the wording in the specification is stable; meaning that all MSCs backing the behaviour changes have finalised, and the only changes at that moment would be fixes to fill oversights.

Consequently, I'm also asking to change the wording of "Switch to the current stable release." to "Switch to the latest release", as that makes more sense.

The definition of stable and unstable is pointed to a lot around the spec, specifically around the proposal process, where an MSC is "stable" once it has passed FCP. So, confusion can occur if someone's "stable" MSC enters an "unstable" specification, is their MSC "unstable" by extension as well? The bending of the definition may be unclear or not easily inferred, as the exact range of "stability" guaranteed by the "stable" definition changes a lot around the spec this way.

I'm aware that the spec can release sub-releases, such as `v1.2-alpha` and such, but even then, i still think "released"/"unreleased" fits this dynamic better than "stable"/"unstable".

**Comment by turt2live** [comment]
We deliberately maintain the ability to change it any time, so it is unstable. It's also true that it's unreleased but I'm not convinced that's the correct word to use.

**Comment by ShadowJonathan** [comment]
"unreleased" infers a definition of "unstable", while not using the exact word. I think it'll help a lot to differentiate like that (and possibly explain the nuance in that same box), so that the language around the spec can become clearer, and with fewer word "collisions"

**Comment by turt2live** [comment]
I agree that it probably needs a better descriptor for what it is, but "unreleased" is uncomfortably too relaxed is all.
